pkg/reconciler/v1alpha1/pipeline/resources/dag.go

```diff
@@ -93,6 +93,58 @@ func (g *DAG) GetPreviousTasks(pt string) []v1alpha1.PipelineTask {
 	return v.getPrevTasks()
 }
 
+// GetSchedulable returns a list of PipelineTask that can be scheduled,
+// given a list of "done" task.
```

Comment:
can you eleaborate here on what a 'list of done tasks' is? (i think `t` is a list of pipeline task names that have completed - successfully?)

Reply:
Yes, indeed, it will be successful tasks here (only) as if one fails, the pipeline will be marked failed and the taskruns too. I should also rename the variables :sweat_smile: